app/scripts/lib/setupSentry.js

```diff
@@ -321,17 +359,15 @@ function hideUrlIfNotInternal(url) {
  */
 export function beforeBreadcrumb(getState) {
   return (breadcrumb) => {
-    if (getState) {
-      const appState = getState();
-      if (
-        Object.values(appState).length &&
-        (!appState?.store?.metamask?.participateInMetaMetrics ||
-          !appState?.store?.metamask?.completedOnboarding ||
-          breadcrumb?.category === 'ui.input')
-      ) {
-        return null;
-      }
-    } else {
+    if (!getState) {
+      return null;
+    }
+    const appState = getState();
+    if (
+      !getMetaMetricsEnabledFromAppState(appState) ||
+      !appState?.store?.metamask?.completedOnboarding ||
```

Comment:
Do we need to handle the case where `appState.store` is undefined, but`appState.persistedState` is defined?

Reply:
Yep, just found this while testing too. Fixed here I think: aabb66be35 (still testing)